test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest
+pytest-mock
+responses

```

Comment:
I did not end up using `reponses` as I considered that an edge which I considered a boundary. Additional testing will need to be performed on this module.